app/templates/base-flat.jade

```diff
@@ -146,123 +146,122 @@ mixin accountLinks
 
   block footer
     if !me.hideFooter()
-      .container-fluid
```

Comment:
File lgtm. Checking that I am reading this correctly. We are removing the wrapping `container-fluid` div.

Reply:
And wanted to clarify scope as well. This is a large change, but scoped to the footer. On manual testing looks great!

Reply:
Yup just moving the wrapped .container-fluid, bootstrap actually doesn't support nesting containers within containers, it can cause render issues in some situations.  In this case the fluid container wasn't doing anything for us. Scoped just to the footer.